plugins/utils/neorg.nix

```diff
@@ -80,7 +80,7 @@ with lib;
           }
         ) modes;
 
-        floatPrecision = helpers.defaultNullOpts.mkNullable types.float "0.01" ''
+        floatPrecision = helpers.defaultNullOpts.mkNullable types.float (literalExpression "0.01") ''
```

Comment:
```suggestion
        floatPrecision = helpers.defaultNullOpts.mkNullable types.float 0.01 ''
```
Why not ?

Reply:
I did it to avoid stylix reformatting with exponent notation. But that doesn't matter because we're passing a value anyawy, so I'll revert.